packages/jaeger-ui/src/components/SearchTracePage/SearchForm.jsx

```diff
@@ -342,6 +342,17 @@ export class SearchFormImpl extends React.PureComponent {
                     <li>
                       Values containing whitespace or equal-sign &apos=&apos should be enclosed in quotes
                     </li>
+                    <li>
+                      Elasticsearch/OpenSearch storage supports regexp query, therefore{' '}
+                      <a
+                        href="https://lucene.apache.org/core/9_0_0/core/org/apache/lucene/util/automaton/RegExp.html"
+                        rel="noopener noreferrer"
+                        target="_blank"
+                      >
+                        reserved characters
+                      </a>{' '}
+                      need to be escaped for exact match queries.
```

Comment:
Should it include an example?

Reply:
I am not sure how to render the example. There is already one example (not for ES). If ES specific examples are added, we should let users know that they are for ES  and I am not sure how to do it.

Reply:
I would just go with the current PR.